### PR TITLE
Root page

### PR DESCRIPTION
**Pull request by mikeheft**
Additions: 
Added carousel

User Story:
3

Concerns (code location):
Getting a js error but working on it. PUshing this to get everyone on same page for now.

Mentions to collaborators:
@amhursh @blsrofe @JunePaloma 

Notes(if applicable):

### CODE REVIEW

File: app/models/category.rb [hunk 1]
```diff
@@ -1,5 +1,14 @@
...
+  validates :name, presence: :true, uniqueness: :true
+
+  def image
+    items.select('items.*')
+      .where("items.category_id = #{self.id}")
+      .order('RANDOM()')
+      .limit(1)
+      .first
+      .image_path
+  end
```
Comment (blsrofe) [inline app/models/category.rb:13]: This is cool!

**Review by blsrofe — APPROVED** [review]

File: app/models/category.rb (hunk 1 above)
Comment (amhursh) [inline app/models/category.rb:13]: Looks like a great application

File: spec/factories/stores.rb [hunk 2]
```diff
@@ -0,0 +1,7 @@
+FactoryGirl.define do
+  factory :store do
+    name "MyString"
+    slug "MyString"
+    description "MyString"
+  end
+end
```
Comment (amhursh) [inline spec/factories/stores.rb:7]: We can make this a smart factory later.

**Review by amhursh — APPROVED** [review]
Looks good. Ready to merge

**Pull request merged**